### PR TITLE
margin/multirow params

### PR DESCRIPTION
**Pull request by fnwaohaHC**
This branch adds parameters for table margin and multi row creation. Depending on the values passed to the GeneratedTable constructor it will output a table with merged cell rows and margin to the top and bottom of the table.

### CODE REVIEW

File: generated_tables/e8PFJcnZF4eCtGoNudqmSz.pdf
Comment (pgaviganHC) [inline generated_tables/e8PFJcnZF4eCtGoNudqmSz.pdf:1]: This generated file should not be in the repository

File: table_processing/GeneratedTable.py [hunk 1]
```diff
@@ -28,25 +37,66 @@ def generate_df(self,rows):
...
-                    #table.add_hline()
+   
+
+                self.filename = sd.uuid()
+
+                self.path = './generated_tables/' + self.filename 
+
+                doc.generate_pdf(self.path, compiler='pdflatex')
+                self.path += '.pdf'
+
```
Comment (pgaviganHC) [inline table_processing/GeneratedTable.py:68]: I am concerned that this gets to be a very deep level of nested looping and if statements. When this happens it tends to point to a need to refactor, perhaps breaking up the logic into multiple functions or other.

**Comment by pgaviganHC** [comment]
Could a description please be provided? I am unsure what the expected outcome of the new code is supposed to be.

Also, I suspect a review by @Louis-Boulet-HC would be a good idea as I am under the impression that this code will impact the pipeline that they have developed.

File: generated_tables/e8PFJcnZF4eCtGoNudqmSz.pdf
Comment (fnwaohaHC) [reply generated_tables/e8PFJcnZF4eCtGoNudqmSz.pdf:1]: deleted

**Comment by fnwaohaHC** [comment]
Added a description

**Review by Louis-Boulet-HC — APPROVED** [review]

**Pull request merged**